cmake/GoogleCloudCppFeatures.cmake

```diff
@@ -23,6 +23,8 @@ include(CreateBazelConfig)
 set(GOOGLE_CLOUD_CPP_LEGACY_FEATURES
     "bigtable;bigquery;iam;logging;pubsub;spanner;storage")
 
+set(GOOGLE_CLOUD_CPP_REST_ONLY_FEATURES "storage;experimental-bigquery_rest")
```

Comment:
Maybe leave a comment explaining why `compute` and `sqladmin` cannot be in that list?

Reply:
Done.